src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -1,12 +1,14 @@
 package com.minecolonies.coremod.util;
 
 import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.blocks.BlockTimberFrame;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.blocks.BlockTimberFrame'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)